.github/workflows/release-notes.yaml

```
@@ -0,0 +1,75 @@
+
+      - name: Update version in config.yaml
+        run: |
+          VERSION=${{ env.VERSION }}
+          sed -i "s/version: .*/version: \"$VERSION\"/" ${{ github.workspace }}/ebusd/config.yaml
+          
+      - name: Configure git
+        run: |
+          git config --global user.email "actions@github.com"
+          git config --global user.name "GitHub Actions"
+
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)